src/bin/clapper-app/clapper-app-uri-dialog.c

```diff
@@ -89,19 +86,17 @@ clapper_app_uri_dialog_open_uri (GtkApplication *gtk_app)
 {
   GtkWindow *window = gtk_application_get_active_window (gtk_app);
   GtkBuilder *builder;
-  AdwMessageDialog *dialog;
+  AdwAlertDialog *dialog;
   GtkWidget *extra_child;
```

Comment:
```suggestion
  GtkWidget *entry_row;
```

Reply:
Ideally, all places that referred to "extra_child" property that now actually use "entry_row" should be updated. I am afraid that leaving naming of something that is "a child of a child of extra_child" with name left as "extra_child" can lead to all kind of issues in the future (wrong assumptions) without looking closely at the UI template.

Sorry for being picky :bowing_man:

Reply:
Done! And no worries, I'm very unfamiliar with the codebase and coding style, thanks for the guidance!